tests/modules/devices/link-unlink-asset.test.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ApiDeviceLinkAssetRequest,
+  ApiDeviceLinkAssetResult,
+} from "../../../index";
+
+import { beforeEachTruncateCollections } from "../../hooks/collections";
+import { beforeAllCreateEngines } from "../../hooks/engines";
+import { beforeEachLoadFixtures } from "../../hooks/fixtures";
+
+import { useSdk } from "../../helpers/sdk";
+
+jest.setTimeout(10000);
+
+describe("DeviceController: receiveMeasure", () => {
+  const sdk = useSdk();
+
+  beforeAll(async () => {
+    await sdk.connect();
+    await beforeAllCreateEngines(sdk);
+  });
+
+  beforeEach(async () => {
+    await beforeEachTruncateCollections(sdk);
+    await beforeEachLoadFixtures(sdk);
+  });
+
+  afterAll(async () => {
+    sdk.disconnect();
+  });
+
+  it("should throw an error if no measures are selected", async () => {
```

Comment:
can you add a test that check the case with one measure name and implicitMeasuresLinking to true ?

Reply:
They already exists in Cucumber. And now they have been migrated in this PR https://github.com/kuzzleio/kuzzle-device-manager/pull/274

Reply:
Actually they existed in Cucumber. And now they have been migrated into Jest in this PR https://github.com/kuzzleio/kuzzle-device-manager/pull/274